src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -99,6 +99,62 @@
             anti_template=r"^(" + r"|".join(dialect.sets("reserved_keywords")) + r")$",
         )
     ),
+    PostFunctionGrammar=AnyNumberOf(
+        Ref("WithinGroupClauseSegment"),
+        Ref("FilterClauseGrammar"),
+        Ref("OverClauseSegment", optional=True),
+    ),
+    FunctionContentsGrammar=AnyNumberOf(
```

Comment:
I think you are just adding a little bit to ansi `PostFunctionGrammar` and `FunctionContentsGrammar`. Instead of copying it all again, let's try to adapt code like this:

```
    LiteralGrammar=ansi_dialect.get_grammar("LiteralGrammar").copy(
        insert=[
            Ref("ReferencedVariableNameSegment"),
        ]
    ),
```

Reply:
Thanks for the tip!
The copy works great with `FunctionContentsGrammar`, but it doesn't work with `PostFunctionGrammar` because ansi_dialect uses `OneOf` and oracle_dialect should use `AnyNumberOf`.
Is there a way to copy the grammar and change the class `OneOf` to `AnyNumberOf`?

Reply:
Ah nice, let’s just copy-paste the 5-line function then, as long as we can avoid copy-pasting the long one. 👍